openstack/compute/v2/servers/testing/requests_test.go

```diff
@@ -171,6 +171,7 @@ func TestCreateServerWithUserdataEncoded(t *testing.T) {
 }
 
 func TestCreateServerWithImageNameAndFlavorName(t *testing.T) {
+	t.Skip("IMS endpoint changed")
```

Comment:
still needed?

Reply:
I'd like to remove `TestCreateServerWithImageNameAndFlavorName`.